### PR TITLE
Tweaks and restructure

### PR DESCRIPTION
**Pull request by Beanow**
Mainly this makes the README your entrypoint and index.
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
+
+# List of roles
+
+Workgroup | Role | Summary
+-|-|-
+[Organization][wg-organization]|`credentials`|People who handle login details / credentials for any part of the organization and share it with the appropriate other workgroups and members.
+[Organization][wg-organization]|`workgroups`|People who review applications for and generally facilitate the operation of workgroups.
+[Organization][wg-organization]|`discord`|Have admin permissions for the Discord server.
+[Organization][wg-organization]|`github`|Have admin permissions for the GitHub organization.
+[Outreach][wg-outreach]|`hello`|Handeling the hello@... e-mail address.
```
Comment (nothingismagick) [inline README.md]: `Managing`, not Handeling

File: README.md [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
+<!-- GENERATED FILE, DON'T EDIT -->
+# Team Open - Workgroup list
+
+[![Get involved](get-involved.svg)](https://github.com/teamopen-dev/workgroups/issues/new?template=wg-join-leave.md&title=Would%20like%20to%20join)
+&nbsp; <sup>[More info on workgroups](ABOUT.md)</sup>
+
+Title | Summary | Members
+-|-|-:
+[Organization][wg-organization]|Organization level day-to-day tasks.|2
+[Outreach][wg-outreach]|Reaching out to where Team Open may help.|2
```
Comment (nothingismagick) [inline README.md]: Reaching out to places and people where Team Open maybe of  help.

File: README.md [hunk 3]
```diff
@@ -1,28 +1,40 @@
...
+[Lifeguard][wg-lifeguard]|Development of the Lifeguard project itself.|2
+[Lifeguard Devs][wg-lifeguard-devs]|People who maintain OSS in the Lifeguard programe.|2
+[Media][wg-media]|(Social) media, publication and communication tasks.|2
+
+# List of roles
+
+Workgroup | Role | Summary
+-|-|-
+[Organization][wg-organization]|`credentials`|People who handle login details / credentials for any part of the organization and share it with the appropriate other workgroups and members.
+[Organization][wg-organization]|`workgroups`|People who review applications for and generally facilitate the operation of workgroups.
```
Comment (nothingismagick) [inline README.md]: People who review membership applications and generally facilitate the operation of workgroups.

File: scripts/main.js [hunk 4]
```diff
@@ -18,22 +18,26 @@ for(const key in workgroups) {
 	const doc = Templates.workgroup(wg);
 	writeFileSync(joinPath(docsPath, `${wg.key}.md`), doc);
```
Comment (nothingismagick) [inline scripts/main.js]: Are you sure this path exists?

File: scripts/main.js [hunk 5]
```diff
@@ -18,22 +18,26 @@ for(const key in workgroups) {
...
+writeFileSync(
+	joinPath(templPath, `wg-0-join-leave.md`),
+	Templates.joinLeaveWgIssue({workgroups, baseUrl: WORKGROUPS_BASE_URL})
+);
+let issueTemplateIndex = 1;
 for(const key in workgroups) {
 	const wg = workgroups[key];
 	const doc = Templates.notifyWgIssue(wg);
-	writeFileSync(joinPath(templPath, `${wg.key}.md`), doc);
+	writeFileSync(joinPath(templPath, `wg-${issueTemplateIndex}-${wg.key}.md`), doc);
```
Comment (nothingismagick) [inline scripts/main.js]: Same as above. Best to ensure it.

File: README.md (hunk 3 above)
Comment (Beanow) [reply README.md]: Actually `membership applications` isn't right. It's reviewing proposals for new workgroups.

File: scripts/main.js (hunk 4 above)
Comment (Beanow) [reply scripts/main.js]: Actually above we just cleared it with rimraf. I can do a mkdirp if you like :]

File: README.md (hunk 3 above)
Comment (Beanow) [reply README.md]: People who review new workgroup proposals and generally facilitate the operation of workgroups.?

Comment (Beanow) [reply README.md]: Or just leave out that detail and keep it to:
People who facilitate the operation of workgroups.

**Comment by Beanow** [comment]
Applied the suggested rephrasing.
I do suggest we leave code refactoring for a later moment. There's a lot to improve there and I probably want to split this off into something you can use as a template later.

**Pull request merged**